transcendence-app/src/users/users.module.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from '@nestjs/common';
+import { UsersService } from './users.service';
+
+@Module({
+  providers: [UsersService],
+  exports: [UsersService],
+})
+export class UsersModule {}
```

Comment:
Check `user` module because you have inconsistent naming in your files, some things are named `user` while others `users`. I'd go with `user` (singular) since we can have one or many of them

Reply:
Done